Profiles - ZIB 2017/zib-Pregnancy-Gravidity.xml

```diff
@@ -0,0 +1,105 @@
+  <copyright value="CC0" />
+  <fhirVersion value="3.0.1" />
+  <mapping>
+    <identity value="hcim-pregnancy-v3.1-2017EN" />
+    <uri value="https://zibs.nl/wiki/Pregnancy-v3.1(2017EN)" />
+    <name value="HCIM Pregnancy-v3.1(2017EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="Observation" />
+  <baseDefinition value="http://fhir.nl/fhir/StructureDefinition/nl-core-observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation.extension">
+      <path value="Observation.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.extension:zib-Pregnancy">
+      <path value="Observation.extension" />
+      <sliceName value="zib-Pregnancy" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/event-partOf" />
+      </type>
+    </element>
+    <element id="Observation.extension:zib-Pregnancy.valueReference:valueReference">
+      <path value="Observation.extension.valueReference" />
+      <sliceName value="valueReference" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-Pregnancy" />
+      </type>
+    </element>
+    <element id="Observation.code.coding">
+      <path value="Observation.code.coding" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="code" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="1" />
+    </element>
+    <element id="Observation.code.coding:Gravidity">
+      <path value="Observation.code.coding" />
+      <sliceName value="Gravidity" />
+      <min value="1" />
+      <max value="1" />
+    </element>
+    <element id="Observation.code.coding:Gravidity.system">
+      <path value="Observation.code.coding.system" />
+      <min value="1" />
+      <fixedUri value="http://loinc.org" />
+    </element>
+    <element id="Observation.code.coding:Gravidity.code">
+      <path value="Observation.code.coding.code" />
+      <min value="1" />
+      <fixedCode value="11996-6" />
+    </element>
+    <element id="Observation.code.coding:Gravidity.display">
+      <path value="Observation.code.coding.display" />
+      <defaultValueString value="Parity" />
+    </element>
+    <element id="Observation.value[x]:valueQuantity">
+      <path value="Observation.valueQuantity" />
+      <sliceName value="valueQuantity" />
```

Comment:
missing concept definition and short name ect.

Reply:
Shame on me